Source: Timo Gurr <tgurr@exherbo.org>
Upstream: TODO
Reason: Fix install location of cmake files

diff -Naur ortp-1.0.2/CMakeLists.txt ortp-1.0.2.new/CMakeLists.txt
--- ortp-1.0.2/CMakeLists.txt	2017-07-20 13:35:30.000000000 +0200
+++ ortp-1.0.2.new/CMakeLists.txt	2018-05-03 19:40:18.924025237 +0200
@@ -206,7 +206,7 @@
 	@ONLY
 )
 
-set(CONFIG_PACKAGE_LOCATION "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CONFIG_PACKAGE_LOCATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 install(EXPORT ${EXPORT_TARGETS_NAME}Targets
 	FILE ORTPTargets.cmake
 	DESTINATION ${CONFIG_PACKAGE_LOCATION}
